tensorflow/python/framework/ops_test.py

```diff
@@ -1308,6 +1308,18 @@ def testBasics(self):
     self.assertEqual("bar_2", g.unique_name("bar", mark_as_used=False))
     self.assertEqual("bar_2", g.unique_name("bar"))
 
+  def testBackslashRegex(self):
+    # GitHub issue 39019, all should pass
+    g = ops.Graph()
+    with g.name_scope("n_CatCntc_campaign\c_campaign"):
```

Comment:
I think we need to escape this backslash, otherwise Python will complain that "\c" is an invalid escape sequence.

Reply:
Thanks @mdanatg, updated.